shared/package.json

```diff
@@ -138,15 +138,15 @@
     "prettier": "1.12.1",
     "prop-types": "15.6.1",
     "purepack": "1.0.4",
-    "qrcode-generator": "1.3.1",
+    "qrcode-generator": "git://github.com/keybase/qrcode-generator#keybase-changes-off-140",
     "re-reselect": "2.1.0",
     "react": "16.4.0",
     "react-dom": "16.4.0",
     "react-gateway": "3.0.0",
     "react-list": "0.8.9",
     "react-measure": "^2.0.0",
     "react-native": "git://github.com/keybase/react-native#keybase-fixes-off-543",
-    "react-native-camera": "0.10.0",
+    "react-native-camera": "1.1.4",
```

Comment:
this was crazy old but was afraid to touch it until now. required updates to gradle (needed for  RN 0.56 anyways)